<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="lb_statusConnect">
      <property name="text">
       <string>Отключено</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTableView" name="tv_result"/>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="gb_filters">
      <property name="title">
       <string>Фильтры</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lb_airport">
         <property name="text">
          <string>Введите аэропорт</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_airport"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_date">
         <property name="text">
          <string>Введите дату</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="de_date"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="pb_departure">
         <property name="text">
          <string>Данные по вылетающим самолетам</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="pb_airports">
         <property name="text">
          <string>Список аэропортов</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="pb_clear">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="pb_statistic">
         <property name="text">
          <string>Статистика</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="pb_arrival">
         <property name="text">
          <string>Данные по прибывающим самолетам</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Подключение</string>
    </property>
    <addaction name="act_addData"/>
    <addaction name="act_connect"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_addData">
   <property name="text">
    <string>Ввести данные</string>
   </property>
  </action>
  <action name="act_connect">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
